config/config.exs

```diff
@@ -63,4 +63,12 @@ config :rollbax, enabled: false
 
 config :logger, :console, format: "[$level] $metadata$message\n"
 
+config :hexpm, Hexpm.OAuthProviders.GitHub,
+  scope: "user:email",
+  redirect_uri: "https://localhost:4000/oauth-login",
+  client_id: "Iv1.3fb12a79b1606f04",
```

Comment:
are these values we should put into ENV VARs to secure them?

Reply:
The only secret value is `client_secret`, but I'm gonna put them all on env vars :)